docs/internals/frontends.rst

```diff
@@ -209,8 +209,10 @@ Standard output
 *stdout* is different and more command-dependent than logging. Commands like :ref:`borg_info`, :ref:`borg_create`
 and :ref:`borg_list` implement a ``--json`` option which turns their regular output into a single JSON object.
 
-Dates are formatted according to ISO-8601 with the strftime format string '%a, %Y-%m-%d %H:%M:%S',
-e.g. *Sat, 2016-02-25 23:50:06*.
+Dates are formatted according to ISO-8601 in local time. Neither an explicit time zone nor microseconds
+are specified *at this time* (subject to change). The equivalent strftime format string is '%Y-%m-%dT%H:%M:%S',
+e.g. 2017-08-07T12:27:20. Standard formatting is used for microseconds, if available. Only :ref:`borg_list`
```

Comment:
"Standard formatting is used for microseconds, if available."

That reads a bit strange, after saying "Neither an explicit time zone nor microseconds are specified *at this time* (subject to change)."

So, what shall it mean?